### PR TITLE
Bugfix/help window minimized bug

### LINKED ISSUE #80: Fix minimized window bug
Modify the Help Window logic so that if it is already open but minimized, the window is restored to its normal state when the help command is executed again. This will eliminate the need for manual restoration of the window by the user.

### PR DESCRIPTION
**Pull request by stanleytangzh**
closes #80

### CODE REVIEW

**Review by dominic2412 — COMMENTED** [review]
It is good that you are actively adding changes to fix the bug, Good work stanley!

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 1]
```diff
@@ -50,7 +50,7 @@ public HelpWindow() {
     }
 
     /**
-     * Shows the help window.
+     * Shows the help window. If the window is minimized, restore it.
```
Comment (thisisaditya17) [inline src/main/java/seedu/address/ui/HelpWindow.java:53]: Great work changing java docs.
Great attention to detail.

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 2]
```diff
@@ -69,8 +69,12 @@ public HelpWindow() {
      */
     public void show() {
         logger.fine("Showing help page about the application.");
-        getRoot().show();
-        getRoot().centerOnScreen();
+        Stage stage = getRoot();
+        if (stage.isIconified()) {
+            stage.setIconified(false); // Restore if minimized
```
Comment (thisisaditya17) [inline src/main/java/seedu/address/ui/HelpWindow.java:74]: The logic seems good

**Review by thisisaditya17 — COMMENTED** [review]
Overall great work fixing this

**Pull request merged**